packages/react-reconciler/src/ReactFiber.js

```diff
@@ -230,8 +230,8 @@ function FiberNode(
   if (enableProfilerTimer) {
     this.actualDuration = 0;
     this.actualStartTime = 0;
```

Comment:
Is this one (`actualStartTime`) intentionally still called "time"? Or would it be more consistent to rename it too?

If it's intentionally still "time", would you say "time" now *always* refers to timestamps and "duration" *always* means length? Or is it still different?

Reply:
Start time and commit time are accurate. They both contain `performance.now()` values. Actual and base durations contain time deltas.

Reply:
So yes, "time" now always means a point in time and "duration" always means a length of time.